doc/dnf5.conf.5.rst

```diff
@@ -0,0 +1,1468 @@
+    command will be skipped without raising the error causing the whole operation to fail.
+    Currently works for install command only.
+
+    Default: ``True``.
+
+.. _tsflags_options-label:
+
+``tsflags``
+    :ref:`list <list-label>`
+
+    List of strings adding extra flags for the RPM transaction.
+
+    ================  ===============================
+    tsflag value      RPM Transaction Flag
+    ================  ===============================
+    ``noscripts``     ``RPMTRANS_FLAG_NOSCRIPTS``
+    ``test``          ``RPMTRANS_FLAG_TEST``
+    ``notriggers``    ``RPMTRANS_FLAG_NOTRIGGERS``
+    ``nodocs``        ``RPMTRANS_FLAG_NODOCS``
+    ``justdb``        ``RPMTRANS_FLAG_JUSTDB``
+    ``nocontexts``    ``RPMTRANS_FLAG_NOCONTEXTS``
+    ``nocaps``        ``RPMTRANS_FLAG_NOCAPS``
+    ``nocrypto``      ``RPMTRANS_FLAG_NOFILEDIGEST``
+    ================  ===============================
+
+    The ``nocrypto`` option will also set the ``_RPMVSF_NOSIGNATURES`` and
+    ``_RPMVSF_NODIGESTS`` VS flags.
+
+    The ``test`` option provides a transaction check without performing the transaction.
+    It includes downloading of packages, gpg keys check (including permanent import of
+    additional keys if necessary), and rpm check to prevent file conflicts.
+
+    The ``nocaps`` is supported with rpm-4.14 or later. When ``nocaps`` is used but rpm
+    doesn't support it, DNF5 only reports it as an invalid tsflag.
+
+    Default: empty.
+
+.. _upgrade_group_objects_upgrade_options-label:
+
+``upgrade_group_objects_upgrade``
+    :ref:`boolean <boolean-label>`
+
+    If enabled, performs the automatic running of ``group upgrade`` when running the ``upgrade`` command.
+
+    Default: ``True``.
+
+.. _varsdir_options-label:
+
+``varsdir``
+    :ref:`list <list-label>`
+
+    List of directories where variables definition files are looked for.
+
+    See :ref:`variable files <varfiles-label>` in Configuration reference.
+
+    Default: ``/etc/dnf/vars``.
+
+.. _zchunk_options-label:
+
+``zchunk``
+    :ref:`boolean <boolean-label>`
+
+    If enabled, repository metadata are compressed using the zchunk format (if available).
+
+    Default: ``True``.
+
+.. _system_state_options-label:
+
+``system_state``
+    TODO documented in dnf5, not dnf4
+
+.. _transaction_history_dir_options-label:
+
+``transaction_history_dir``
+    TODO documented in dnf5, not dnf4
+
+.. _transformdb_options-label:
+
+``transformdb``
+    TODO documented in dnf5, not dnf4
+
+.. _recent_options-label:
+
+``recent``
+    TODO documented in dnf5, not dnf4
+
+.. _reset_nice_options-label:
+
+``reset_nice``
+    TODO documented in dnf5, not dnf4
+
+.. _system_cachedir_options-label:
+
+``system_cachedir``
+    TODO documented in dnf5, not dnf4
+
+.. _debugdir_options-label:
+
+``debugdir``
+    TODO documented in dnf5, not dnf4
+
+.. _optional_metadata_types_options-label:
+
+``optional_metadata_types``
```

Comment:
Suggested text:

:ref:`list <list-label>`

List of the following: ``comps``, ``filelists``, ``other``, ``presto``, ``updateinfo``.

Defines which types of metadata are to be loaded in addition to ``primary`` and ``modules``, which are loaded always as they are essential. Note that the list can be extended by individual commands during runtime.

Default: ``comps,updateinfo``.

Reply:
Although marked as resolved, description for this option is empty in the current version...